hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/TestContainerReportHandlingWithHA.java

```diff
@@ -113,23 +110,7 @@ void testDeletingOrDeletedContainerTransitionsToClosedWhenNonEmptyReplicaIsRepor
         // restart a DN and wait for the container to get CLOSED in all SCMs
         HddsDatanodeService dn = cluster.getHddsDatanode(keyLocation.getPipeline().getFirstNode());
         cluster.restartHddsDatanode(dn.getDatanodeDetails(), false);
-        ContainerManager[] array = new ContainerManager[numSCM];
-        for (int i = 0; i < numSCM; i++) {
-          array[i] = cluster.getStorageContainerManager(i).getContainerManager();
-        }
```

Comment:
Previously the HA test checked container state in all 3 SCMs, but I guess it is enough to check only the leader, as the test does with the patch.